src/__tests__/index.test.ts

```diff
@@ -94,4 +98,19 @@ describe('index', () => {
 
     expect(mtLinkSdk.storedOptions.mode).toBe('production');
   });
+
+  test('chanegs the samlSubjectId when calling setSamlSubjectId', () => {
```

Comment:
```suggestion
  test('change the samlSubjectId when calling setSamlSubjectId', () => {
```
so many typo, did you guys not review your own PR first?